src/main/groovy/com/instabot/Main.groovy

```diff
@@ -27,6 +27,7 @@ class Main implements CommandLineRunner {
     void run(String... args) throws Exception {
         LOG.info("Spring application successfully initialized")
         Date startTime = new Date()
+        // TODO: Just username and password would be enough
```

Comment:
There are different kinds of "username" used across the app, e.g:
- login - to log in to Instagram and get access to data
- master - the user whose followers and the following are extracted
It doesn't have to be the same user. E.g., I can log in to my account to extract your followers and following.
So I decided to append the "login" prefix, to make the difference explicit. 
Please let me know if it makes sense for you.